### DISCUSSION TITLE
Package stability

### DISCUSSION THREAD

**Discussion by garrysanders** [wol-soft/php-json-schema-model-generator#67] [General]
Is this package in a stable state and if not is there any plans to get it to a stable point?

I know this isn't technically tagged as stable (v0.x) but unsure if it's just how this package has been tagged as I can't see any mention of it being in development.

I posted this under issues before noticing this is more appropriate place to ask

**Comment by wol-soft** [comment]
I've started a major rework a while ago with the goal to support multiple JSON Schema versions and the goal to tag a stable 1.0 version. The result wasn't as successful as expected so I'll start a new try when time allows.

Although there might be some issues (eg the issue #65) I would consider the package relatively stable due to the large test suite and my usages of the package in various projects (compare execute tests CI step @ https://github.com/wol-soft/php-json-schema-model-generator/actions/runs/3584287792/jobs/6030765761 and https://coveralls.io/github/wol-soft/php-json-schema-model-generator?branch=master).

Additionally the intended rework will not change the API of the package in a substantial way.

I'll close your issue to keep the stability discussion here ;)
